packages/web-frontend/src/tests/components/dateRangePicker.test.js

```diff
@@ -12,23 +12,67 @@ import { DateRangePicker } from '../../components/DateRangePicker';
 import {
   GRANULARITY_CONFIG,
   GRANULARITIES_WITH_ONE_DATE,
-  formatMomentAsString,
+  momentToDateString,
+  GRANULARITIES,
 } from '../../utils/periodGranularities';
 import { MIN_DATE_PICKER_DATE } from '../../components/DateRangePicker/constants';
 
-const MIN_MOMENT_DATE = moment(MIN_DATE_PICKER_DATE);
 const MAX_MOMENT_DATE = moment();
+
 const START_DATE = '2016-09-23';
 const END_DATE = '2018-03-20';
 
+const MIN_MOMENT_STRINGS = {
+  [GRANULARITIES.DAY]: '1st January 2015',
+  [GRANULARITIES.SINGLE_DAY]: '1st January 2015',
+  [GRANULARITIES.WEEK]: 'W/C 29 Dec 2014',
+  [GRANULARITIES.SINGLE_WEEK]: 'W/C 29 Dec 2014',
+  [GRANULARITIES.MONTH]: 'Jan 2015',
+  [GRANULARITIES.SINGLE_MONTH]: 'Jan 2015',
+  [GRANULARITIES.QUARTER]: 'Q1 2015',
+  [GRANULARITIES.SINGLE_QUARTER]: 'Q1 2015',
+  [GRANULARITIES.YEAR]: '2015',
+  [GRANULARITIES.SINGLE_YEAR]: '2015',
+};
+
+const TEST_START_DATE_STRINGS = {
+  [GRANULARITIES.DAY]: '23rd September 2016',
+  [GRANULARITIES.SINGLE_DAY]: '23rd September 2016',
+  [GRANULARITIES.WEEK]: 'W/C 19 Sep 2016',
+  [GRANULARITIES.SINGLE_WEEK]: 'W/C 19 Sep 2016',
+  [GRANULARITIES.MONTH]: 'Sep 2016',
+  [GRANULARITIES.SINGLE_MONTH]: 'Sep 2016',
+  [GRANULARITIES.QUARTER]: 'Q3 2016',
+  [GRANULARITIES.SINGLE_QUARTER]: 'Q3 2016',
+  [GRANULARITIES.YEAR]: '2016',
+  [GRANULARITIES.SINGLE_YEAR]: '2016',
+};
+
+const TEST_END_DATE_STRINGS = {
+  [GRANULARITIES.DAY]: '20th March 2018',
+  [GRANULARITIES.SINGLE_DAY]: '20th March 2018',
+  [GRANULARITIES.WEEK]: 'W/C 19 Mar 2018',
+  [GRANULARITIES.SINGLE_WEEK]: 'W/C 19 Mar 2018',
+  [GRANULARITIES.MONTH]: 'Mar 2018',
+  [GRANULARITIES.SINGLE_MONTH]: 'Mar 2018',
+  [GRANULARITIES.QUARTER]: 'Q1 2018',
+  [GRANULARITIES.SINGLE_QUARTER]: 'Q1 2018',
+  [GRANULARITIES.YEAR]: '2018',
+  [GRANULARITIES.SINGLE_YEAR]: '2018',
+};
+
 describe('dateRangePicker', () => {
+  it('Has a MIN_DATE_PICKER_DATE consistent with tests', () => {
```

Comment:
This is more so that if it changes and this test fails, the error is easier to diagnose, as opposed to testing any functionality.

Reply:
Good idea!